includes/wc-formatting-functions.php

```diff
@@ -271,11 +271,11 @@ function wc_format_decimal( $number, $dp = false, $trim_zeros = false ) {
 
 	// DP is false - don't use number format, just return a string in our format
 	} elseif ( is_float( $number ) ) {
-		$number = wc_clean( str_replace( $decimals, '.', strval( $number ) ) );
+		$number = strval( $number );
```

Comment:
Not needed here, because $number is float, so there's no char to replace. Only change float type to string.